mmdt/templates/polls/all_results.html

```diff
@@ -0,0 +1,85 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+    {% if is_admin %}
+        <h2>Admin View - All Questions - Voting Results</h2>
+    {% else %}
+        <h2>All Questions - Voting Results</h2>
+    {% endif %}
+    
+    <ul class="nav nav-tabs" id="questionTabs" role="tablist">
+        {% for question in all_questions %}
+            <li class="nav-item">
+                <a class="nav-link {% if forloop.first %}active{% endif %}" id="question-tab-{{ question.id }}" 
+                data-toggle="tab" href="#question-{{ question.id }}" role="tab" aria-controls="question-{{ question.id }}" 
+                aria-selected="{% if forloop.first %}true{% else %}false{% endif %}">{{ question.question_text }}</a>
+            </li>
+        {% endfor %}
+    </ul>
+
+    <div class="tab-content">
+        {% for question in all_questions %}
+            <div class="tab-pane fade {% if forloop.first %}show active{% endif %}" id="question-{{ question.id }}" role="tabpanel" aria-labelledby="question-tab-{{ question.id }}">
+                {% if is_admin %}
+                    <canvas id="resultChart_{{ question.id }}" width="500" height="500"></canvas>
+                    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+
+                    <script>
+                        document.addEventListener('DOMContentLoaded', function () {
```

Comment:
would like to suggest to put the `js` file to separate `js` file and call it from `html. file` by mentioned. that is the best practise for us.